tests/test_script.py

```diff
@@ -520,12 +520,13 @@ async def test_script_callFunctionWithNode_resultReceived(websocket,
     await goto_url(websocket, context_id,
                    "data:text/html,<h2>test</h2>")
 
-    # 1. Get element.`
+    # 1. Get element.
     result = await execute_command(websocket, {
-        "method": "PROTO.browsingContext.findElement",
+        "method": "script.evaluate",
         "params": {
-            "selector": "body > h2",
-            "context": context_id}})
+            "expression": "document.querySelector('body > h2');",
+            "target": {"context": context_id},
+            "awaitPromise": True}})
```

Comment:
This should fix the test. [By default the `handle` is not provided](https://w3c.github.io/webdriver-bidi/#evaluate-function-body).
> 12. Let result ownership be the value of the resultOwnership field of command parameters, if present, or none otherwise.

```suggestion
            "awaitPromise": True,
            "resultOwnership": "root"}})
```

Reply:
Thanks! Confirming it works.